### PR TITLE
Fix supporter tag email translation

### PR DESCRIPTION
**Pull request by notbakaneko**
missing update for #2150

### CODE REVIEW

**Comment by nanaya** [comment]
I think you missed the other one =D

**Comment by notbakaneko** [comment]
(ﾉ °Д°)ﾉ ミ┻━┻

File: app/Models/SupporterTag.php [hunk 1]
```diff
@@ -98,12 +98,19 @@ public static function getMinimumDonation(int $duration)
 
     public static function getDurationText($length)
     {
+        // don't forget to update StoreSupporterTagPrice.durationText in coffee
```
Comment (kj415j45) [inline app/Models/SupporterTag.php]: ~~Don't forget to remove the note~~

File: resources/assets/coffee/_classes/store-supporter-tag-price.coffee [hunk 2]
```diff
@@ -65,6 +65,7 @@ class @StoreSupporterTagPrice
     months: Math.floor(@duration() % 12)
 
   durationText: ->
+    # don't forget to update SupporterTag::getDurationText() in php
```
Comment (kj415j45) [inline resources/assets/coffee/_classes/store-supporter-tag-price.coffee]: ~~Don't forget to remove the note~~

**Pull request merged**